src/reporters/ClearTextReporter.ts

```diff
@@ -1,6 +1,9 @@
 import {Reporter, MutantResult, MutantStatus, SourceFile} from 'stryker-api/report';
 import * as chalk from 'chalk';
 import * as _ from 'lodash';
+import * as log4js from 'log4js';
```

Comment:
By logging mutant results only in the `ClearTextReporter`, we will never see the mutants logged when this reporter is not active. Do you think we should maybe specify that somewhere?